profile/README.md

```diff
@@ -2,14 +2,15 @@
 
 **Self-determined data exchange for everyone** - We create a secure data exchange and storage opportunity to allow companies and education providers to communicate with individuals (e.g. applicants, employees or students) in an encrypted way 🔐
 
-👉 To get up to speed quickly, start with our [User-Guide](https://enmeshed.eu/use/basics)
+👉 To get up to speed quickly, start with our [Integrator-Guide](https://enmeshed.eu/integrate/basics)
 
-👉 You need more business insights? Visit our [Website](https://enmeshed.de), [LinkedIn](https://www.linkedin.com/showcase/enmeshed), [Twitter](https://twitter.com/enmeshed_jssoft) or [Mastodon](https://mastodon.social/@enmeshed)! 
+👉 You need more business insights? Visit our [Website](https://enmeshed.de), [LinkedIn](https://www.linkedin.com/showcase/enmeshed), [Twitter](https://twitter.com/enmeshed_jssoft) or [Mastodon](https://mastodon.social/@enmeshed)!
 
-------------------------------------------------------------------------------------------------------------------------------------------------------------
+---
 
 Harassment-free experience for everyone - We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community --> 🌈 [Contribution guidelines](https://github.com/nmshd/.github/blob/main/CONTRIBUTING.md) + [Code of conduct](https://github.com/nmshd/.github/blob/main/CODE_OF_CONDUCT.md)
 
 Enmeshed takes the security of our software products and services seriously, including all of the open source code repositories managed through our organization --> 🚓 [Security Policy](https://github.com/nmshd/.github/security/policy)
 
-![enmeshed-logo_invertiert klein](https://user-images.githubusercontent.com/128588360/236841580-68334789-71e4-4774-9aa4-08fa7881c2ad.svg)
+![The Enmeshed logo in light theme](./enmeshed-logo-dark.svg#gh-light-mode-only)
```

Comment:
Ich bin etwas verwirrt. Eva meinte mal, dass wir enmeshed überall klein schreiben sollen. Ja, das haben wir in der Doku noch nicht gemacht, aber mich irritiert, dass du es auch groß schreibst. 😄

Reply:
kommt von mir, das ist aber auch nur ein Hilfetext, oder?